Account/index.ts

```diff
@@ -14,8 +16,13 @@ export interface Account extends Account.Creatable {
 	counterparts?: Record<string, Rail.Address>
 	key?: string
 	rules?: Rule[]
+	status: Status
 }
 export namespace Account {
+	export type Legacy = Omit<Account, "status">
+	export function fromLegacy(maybeLegacy: Legacy | Account, status?: Status): Account {
+		return { ...maybeLegacy, status: status ?? ("status" in maybeLegacy ? maybeLegacy.status : { mode: "active" }) }
+	}
```

Comment:
Remember to add history to accounts and if it's not optional we should add it to the fromLegacy function.

Reply:
History is not supposed to be on account!